docs/UserGuide.adoc

```diff
@@ -72,6 +73,10 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sorting contacts in address book and listing all persons : 'sort'
```

Comment:
That's good that you have updated the User Guide (UG) to show the newly implemented function that you have created. The UG is the place where the user will go to find out the different features and the usage of it most of the time.